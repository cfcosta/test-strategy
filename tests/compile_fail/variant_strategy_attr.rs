use test_strategy::Arbitrary;
#[derive(Arbitrary, Debug)]
enum X {
    #[strategy(0..10usize)]
    A(usize),
}

fn main() {}
