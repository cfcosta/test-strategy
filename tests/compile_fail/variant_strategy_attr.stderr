error: `#[strategy]` cannot be specified for a variant. Consider specifying it for a field instead.
 --> tests/compile_fail/variant_strategy_attr.rs:4:5
  |
4 |     #[strategy(0..10usize)]
  |     ^
